inc/parser/xhtml.php

```diff
@@ -1345,7 +1345,10 @@ function tabletbody_close() {
     function tablerow_open() {
         // initialize the cell counter used for classes
         $this->_counter['cell_counter'] = 0;
-        $class                          = 'row'.$this->_counter['row_counter']++;
+		if (!isset($this->_counter['row_counter'])){
+            $this->_counter['row_counter'] = 0;
```

Comment:
At line 1293 in function table_open() the row_counter is already initialized
Does that code nothing? 
Or is that function never called for your specific case?

```php
// initialize the row counter used for classes
         $this->_counter['row_counter'] = 0;
```